src/types.js

```diff
@@ -1,8 +1,11 @@
-let F = require('futil')
+import F from 'futil'
+import * as exampleTypes from './example-types/index.js'
+import pivot from './example-types/metricGroups/pivot.js'
 
-module.exports = (config = {}) =>
-  F.mapValuesIndexed((x, type) => F.callOrReturn(x, config[type]), {
-    ...require('./example-types'),
-    // This can't be done in example-types index because it creates a circular dependency (pivot needs all groupStats types)
```

Comment:
this comment is critical to understand why we aren't just importing pivot in example types (and also we should check if this is still an issue after moving to ESM)

Reply:
Good catch. It must've been lost in the reshuffle. I will add it back.

Reply:
The circular import is still an issue with ESM